monix-eval/shared/src/main/scala/monix/eval/Task.scala

```diff
@@ -1916,6 +1915,42 @@ sealed abstract class Task[+A] extends TaskBinCompat[A] with Serializable {
     */
   final def uncancelable: Task[A] =
     TaskCancellation.uncancelable(this)
+
+  /** Times the `Task` execution and returns its duration and the computed value.
+    *
+    * Basic usage example:
+    *
+    * {{{
+    *   import scala.concurrent.duration.NANOSECONDS
```

Comment:
Why is the import needed in this example?

Reply:
typo. Thanks !